tonic/src/server/grpc.rs

```diff
@@ -95,13 +267,31 @@ where
         S: ClientStreamingService<T::Decode, Response = T::Encode>,
         B: Body + Send + Sync + 'static,
         B::Error: Into<crate::Error> + Send + 'static,
+        T: std::fmt::Debug,
```

Comment:
hmmmm this is new, why?

Reply:
Good catch! Was leftover from some debugging.